pkg/skaffold/deploy/helm.go

```diff
@@ -51,10 +50,13 @@ import (
 
 var (
 	// versionRegex extracts version from "helm version --client", for instance: "2.14.0-rc.2"
-	versionRegex = regexp.MustCompile(`\"v(\d[\w\.\-\.]+)`)
+	versionRegex = regexp.MustCompile(`v(\d[\w.\-]+)`)
 
 	// helm3Version represents the version cut-off for helm3 behavior
 	helm3Version = semver.MustParse("3.0.0-beta.0")
+
+	// error to throw when helm version can't be determined
+	versionErrorString = "failed to determine binary version: %w"
```

Comment:
optional: I personally prefer to see errors repeated for clarity than made into variables, but I don't care enough to demand it here.

Reply:
It was an explicit request from @nkubala